server/server.go

```diff
@@ -481,34 +481,6 @@ func (s *Server) IsReady(ctx context.Context) (bool, error) {
 	return s.datastore.IsReady(ctx)
 }
 
-type AuthenticationOverride struct{}
```

Comment:
Moved this section down to the bottom so it doesn't intercept the other server method declarations.